### PR TITLE
Fixes brave/brave-browser#1229 Webtorrent media scaling.

### PR DESCRIPTION
**Pull request by kylemiller3**
Screenshots of changes:
[https://i.imgur.com/Z58xjWr.png](https://i.imgur.com/Z58xjWr.png)
[https://i.imgur.com/NRpTmFO.png](https://i.imgur.com/NRpTmFO.png)

## Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave/brave-browser/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- Verified that these changes build without errors on
  - [ ] Windows
  - [ ] macOS
  - [x] Linux
- Verified that these changes pass automated tests (`npm test brave_unit_tests && npm test brave_browser_tests && npm run test-security`) on
  - [ ] Windows
  - [ ] macOS
  - [x] Linux
- [x] Verified that all lint errors/warnings are resolved (`npm run lint`)
- [x] Ran `git rebase master` (if needed).
- [x] Ran `git rebase -i` to squash commits (if needed).
- [ ] Tagged reviewers and labelled the pull request as needed.
- [ ] Request a security/privacy review as needed.
- [ ] Add appropriate QA labels (QA/Yes or QA/No) to include the closed issue in milestone
- [ ] Public documentation has been updated as necessary. For instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protection-Mode
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-compatibility-issues-with-tracking-protection

## Test Plan:
Load a video torrent in webtorrent and see it it fills the body of the window in letterbox effect.
Load an image torrent in webtorrent and see the iframe fills the body of the window.

## Reviewer Checklist:

- [ ] New files have MPL-2.0 license header.
- [ ] Request a security/privacy review as needed.
- [ ] Adequate test coverage exists to prevent regressions 
- [ ] Verify test plan is specified in PR before merging to source

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on.
- [ ] All relevant documentation has been updated.

### CODE REVIEW

**Comment by bbondy** [comment]
Is this still valid? CC @feross

**Comment by kylemiller3** [comment]
Should still work. Pretty trivial fixes

File: components/brave_webtorrent/extension/components/mediaViewer.tsx [hunk 1]
```diff
@@ -60,4 +60,20 @@ export default class MediaViewer extends React.PureComponent<Props, {}> {
...
+    if (document.getElementById('video') !== null) {
+      let style = document.body.style
+      style.paddingTop = '0px'
+      style.paddingLeft = '0px'
+      style.paddingBottom = '0px'
+      style.paddingRight = '0px'
+      style.marginTop = '0px'
+      style.marginLeft = '0px'
+      style.marginBottom = '0px'
+      style.marginRight = '0px'
```
Comment (feross) [inline components/brave_webtorrent/extension/components/mediaViewer.tsx]: Is there a reason why these styles cannot be applied using CSS?

```css
video {
  padding: 0;
  margin: 0;
}
```

Comment (kylemiller3) [reply components/brave_webtorrent/extension/components/mediaViewer.tsx]: To be honest it's been so long ago now I don't quite remember if there is a reason or it was just something I overlooked.

**Comment by feross** [comment]
Thanks for the PR! I needed to add support for `<audio>` tags and make some other changes so it  was easier to make a new PR:  https://github.com/brave/brave-core/pull/3005

**Pull request closed without merging**